server/models/connections.go

```diff
@@ -7,6 +7,20 @@ import (
 	"github.com/gofrs/uuid"
 )
 
+// swagger:response ConnectionStatus
+type ConnectionStatus string
+
+const (
+	DISCOVERED 		ConnectionStatus = "discovered"
+	REGISTERED 		ConnectionStatus = "registered"
+	CONNECTED 		ConnectionStatus = "connected"
+	IGNORED 		ConnectionStatus = "ignored"
+	MAINTENANCE 	ConnectionStatus = "maintenance"
+	DISCONNECTED 	ConnectionStatus = "disconnected"
+	DELETED 		ConnectionStatus = "deleted"
+	NOT_FOUND 		ConnectionStatus = "not_found"
```

Comment:
```suggestion
	NOTFOUND 		ConnectionStatus = "not found"
```

Are these string values for purposes of displaying in the UI? If so, let's go ahead and capitalize the start of each word.